### PR TITLE
New platform Microsoft Teams

### PR DESCRIPTION
**Pull request by peroyvind**
**Description:**
Documentation for new platform Microsoft Teams

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27981

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/msteams.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+{% configuration %}
+name:
+  description: Setting this parameter allows multiple notifiers to be created. The notifier will bind to the service `notify.NOTIFIER_NAME`.
+  required: false
+  type: string
+  default: "notify"
+url:
+  description: The webhook url created in the setup step.
+  required: true
+  type: url
```
Comment (frenck) [inline source/_integrations/msteams.markdown]: `url` is not a valid type, use `string` instead.

```suggestion
  type: string
```

File: source/_integrations/msteams.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Microsoft Teams"
+description: "Instructions on how to send notification to a Microsoft Teams channel."
```
Comment (frenck) [inline source/_integrations/msteams.markdown]:
```suggestion
description: "Instructions on how to send a notification to a Microsoft Teams channel."
```

File: source/_integrations/msteams.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ha_category:
+  - Notifications
+ha_release: 0.101
+---
+
+The `Microsoft Teams` platform allows you to send notifications from Home Assistant to a team channel in [Microsoft Teams](https://products.office.com/en-us/microsoft-teams/group-chat-software).
+
+## Setup
+
+To send notification to teams you need add the Incoming Webhook app to your team channel. When the app is added you will receive a webhook url that needs to be added to your `configuration.yaml`.
```
Comment (frenck) [inline source/_integrations/msteams.markdown]:
```suggestion
To send a notification to teams, you need to add the Incoming Webhook app to your team channel. When the app is added, you will receive a webhook URL that needs to be added to your `configuration.yaml`.
```

File: source/_integrations/msteams.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Configuration
+
+To add the Microsoft Teams platform to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+notify:
+  - platform: msteams
+    name: myteam
```
Comment (frenck) [inline source/_integrations/msteams.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/msteams.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+{% configuration %}
+name:
+  description: Setting this parameter allows multiple notifiers to be created. The notifier will bind to the service `notify.NOTIFIER_NAME`.
+  required: false
+  type: string
+  default: "notify"
+url:
+  description: The webhook url created in the setup step.
```
Comment (frenck) [inline source/_integrations/msteams.markdown]:
```suggestion
  description: The webhook URL created in the setup step.
```

File: source/_integrations/msteams.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+| ---------------------- | -------- | ----------- |
+| `image_url`            |      yes | Attach an image to the message.
+
+Example for posting file from URL:
+
+```yaml
+title: Title of the message.
+message: Message that will be added.
+data:
+  image_url: http://[url to file, photo, security camera etc]
```
Comment (frenck) [inline source/_integrations/msteams.markdown]:
```suggestion
  image_url: URL_OF_IMAGE
```

File: source/_integrations/msteams.markdown [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Microsoft Teams"
+description: "Instructions on how to send notification to a Microsoft Teams channel."
+logo: msteams.png
```
Comment (frenck) [inline source/_integrations/msteams.markdown]: Did you add the logo somewhere? It doesn't seem to be in this PR.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @peroyvind!

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

**Comment by peroyvind** [comment]
> Thanks for the PR @peroyvind!
> 
> I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

Thanks! Updated the PR 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by fabaff — APPROVED** [review]

**Pull request merged**